docker/docker-compose.yml

```diff
@@ -1,53 +1,65 @@
-version: '2.1'
+version: '3'
 services:
-  database:
+  postgres:
     image: postgres:latest
-    ports:
-      - "5432:5432"
     environment:
-      POSTGRES_USER: "postgres"
-      POSTGRES_PASSWORD: "postgres"
-      POSTGRES_DB:       "ostock_test"
+      POSTGRES_DB: dockerdb
+      POSTGRES_USER: dockeruser
```

Comment:
leave this as postgres user.  The postgres user is the root user with full permissions, if we create a new user we will have to give them permissions to insert and query data.